src/clients/FlexbaseClient.Invoice.ts

```diff
@@ -2,12 +2,23 @@ import { InvoiceOptions } from '../models/Invoice/InvoiceOptions';
 import { Invoice } from '../models/Invoice/Invoice';
 import { FlexbaseClientBase } from './FlexbaseClient.Base';
 
-interface InvoiceResponse {
+interface InvoicesResponse {
     success: boolean;
     error?: string;
     invoices: Invoice[];
 }
 
+interface InvoiceResponse {
```

Comment:
there's a `FlexbaseResponse` that you can inherit so you don't need the success or error fields. Minor thing

Reply:
go it!